deps.windows/20-freetype.ps1

```diff
@@ -0,0 +1,94 @@
+param(
+    [string] $Name = 'freetype',
+    [System.Version] $Version = '2.10.4',
+    [string] $Uri = 'https://github.com/freetype/freetype.git',
+    [string] $Hash = '6a2b3e4007e794bfc6c91030d0ed987f925164a8'
+)
+
+function Setup {
+    Setup-Dependency -Uri $Uri -Hash $Hash -DestinationPath $Path
+}
+
+function Clean {
+    Set-Location $Path
+    if ( Test-Path "build_${Target}" ) {
+        Log-Information "Clean build directory (${Target})"
+        Remove-Item -Path "build_${Target}" -Recurse -Force
+    }
+}
+
+function Configure {
+    Log-Information "Configure (${Target})"
+    Set-Location $Path
+
+    $OnOff = @('OFF', 'ON')
+    $Options = @(
+        $CmakeOptions
+        "-DBUILD_SHARED_LIBS=$($OnOff[$script:Shared.isPresent])"
+        '-DFT_WITH_BROTLI=OFF'
+        '-DFT_WITH_BZIP2=OFF'
+        '-DFT_WITH_HARFBUZZ=OFF'
+        '-DFT_WITH_PNG=OFF'
+        '-DFT_WITH_ZLIB=OFF'
```

Comment:
I actually tried to disable the spam generated by CMake because it cannot find any of these on Windows - it seems it still tries to find the libraries (even if I explicitly disable them), but I also tried to be explicit about disabling anything we don't need.

Reply:
Fair enough. Maybe FreeType's CMake for these needs some work? Something to look into another time.